app/src/main/java/com/example/learingrealmandretrofit/objects/DeckRealm.kt

```diff
@@ -15,4 +15,4 @@ open class DeckRealm(
     override fun toString(): String {
         return Gson().toJson(this)
     }
-}
\ No newline at end of file
+}
```

Comment:
It should be called `Deck`.

Reply:
I have model Deck for request. How right to name DeckRealm? 
https://github.com/osmovia/LearingRealmRetrofit/blob/main/app/src/main/java/com/example/learingrealmandretrofit/objects/Deck.kt

Reply:
My comment is the same, you should use `Deck` for Realm objects, the other class should be renamed to something else.